staging/src/k8s.io/component-base/cli/flag/ciphersuites_flag_test.go

```diff
@@ -86,7 +86,8 @@ func TestConstantMaps(t *testing.T) {
 		if strings.HasPrefix(declName, "VersionTLS") {
 			discoveredVersions[declName] = true
 		}
-		if strings.HasPrefix(declName, "TLS_RSA_") || strings.HasPrefix(declName, "TLS_ECDHE_") {
+		if strings.HasPrefix(declName, "TLS_RSA_") || strings.HasPrefix(declName, "TLS_ECDHE_") || 
+		strings.HasPrefix(declName, "TLS_AES_") || strings.HasPrefix(declName, "TLS_CHACHA20_") {
```

Comment:
@liggitt I added the prefix of TLS1.3 ciphers to ensure the test will cover them going forward.